charts/opensearch/CHANGELOG.md

```diff
@@ -13,7 +13,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
-
+## [1.5.9]
```

Comment:
@DandyDeveloper @peterzhuamazon Should we bump a minor version here?

Reply:
This PR assigned 1.5.9 and will merge once 1.5.8 merged.

Reply:
@DandyDeveloper @peterzhuamazon What I meant was should we make it 1.6.0 instead?